data/src/main/java/org/apache/iceberg/data/IcebergGenerics.java

```diff
@@ -65,8 +65,8 @@ public ScanBuilder caseInsensitive() {
       return this;
     }
 
-    public ScanBuilder select(String... columns) {
-      this.columns = ImmutableList.copyOf(columns);
+    public ScanBuilder select(String... selectColumns) {
```

Comment:
What about `selectedColumns`? I believe it is frequently used throughout the project.

Reply:
Will do